recipes-domx/meta-xt-images-vgpu/recipes-graphics/gles-module/gles-user-module_1.9.bb

```diff
@@ -0,0 +1,12 @@
+require gles-common.inc
+require gles-user-module.inc
+
+DEPENDS += " \
+    clang-native \
+    wayland-protocols \
+"
+
+PREFERRED_VERSION_gles-module-egl-headers = "1.9"
```

Comment:
do we need this?
if so, pls put a comment why this recipe, which has all the headers. still needs headers to be installed before